akka-http-tests/src/test/scala/akka/http/scaladsl/marshalling/ContentNegotiationSpec.scala

```diff
@@ -136,6 +137,20 @@ class ContentNegotiationSpec extends FreeSpec with Matchers {
       accept(`text/plain`, `text/xml`) should select(`text/plain` withCharset `UTF-8`)
       accept(`text/xml`, `text/plain`) should select(`text/xml` withCharset `UTF-8`)
     }
+
+    // https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.1
```

Comment:
The currently valid RFC for HTTP/1.1 is this one: https://tools.ietf.org/html/rfc7231#section-5.3.2. Could you update the link and cross-check that everything is still valid?